vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -330,6 +330,11 @@ ${(commandOutputJson as CommandExecutorResult).stderr}.`),
      */
     public async endSudoProcessMaster(eventStream : IEventStream) : Promise<number>
     {
+        if(os.platform() !== 'linux')
+        {
+            return 0;
+        }
```

Comment:
This function is called at the end of any operation (install, uninstall) once it's finished to kill the process. It could alternatively be conditioned to only be called when on linux, but this reduces code duplication